apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -197,6 +197,12 @@ const radioGroupCss = css`
   .sberRadioGroup .ant-radio-button-wrapper-checked:not(.ant-radio-button-wrapper-disabled):hover::before{
     background-color:${colors.black};
   }
+  .sberRadioGroup .ant-radio-inner::after {
```

Comment:
why should we move this styles in global?

Reply:
This is not used now